src/components/IndexTable/tests/IndexTable.test.tsx

```diff
@@ -160,6 +160,17 @@ describe('<IndexTable>', () => {
     });
   });
 
+  it('does not render checkboxes when selectable is set to false', () => {
```

Comment:
Ahh I see, you're testing checkboxes here. There's an argument to be made for unit testing this Row checkbox functionality only within the Row component but it's more down to your testing philosophy 🤷

Reply:
Agreed, moved this and added a couple more tests in `Row.test.tsx` to validate the checkbox presence